modules/sfp_jsonwhois.py

```diff
@@ -0,0 +1,248 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:        sfp_jsonwhois
+# Purpose:     Search JsonWHOIS for WHOIS records associated with a domain.
+#
+# Authors:     <bcoles@gmail.com>
+#
+# Created:     2020-06-20
+# Copyright:   (c) bcoles 2020
+# Licence:     GPL
+# -------------------------------------------------------------------------------
+
+import json
+import time
+import urllib.request, urllib.parse, urllib.error
+from sflib import SpiderFoot, SpiderFootPlugin, SpiderFootEvent
+
+class sfp_jsonwhois(SpiderFootPlugin):
+    """JsonWHOIS:Footprint,Investigate,Passive:Search Engines:apikey:Search JsonWHOIS for WHOIS records associated with a domain."""
```

Comment:
Perhaps rename to JsonWHOIS.com? There is a JsonWHOIS.io too, so may confuse users when obtaining the API key.

Reply:
Done